packages/env/lib/build-docker-compose-config.js

```diff
@@ -237,8 +234,8 @@ module.exports = function buildDockerComposeConfig( config ) {
 				volumes: developmentMounts,
 			},
 			'tests-wordpress': {
+				build: '.',
```

Comment:
Does this break how `testsPhpVersion` sets the PHP version for the `tests-wordpress` environment?

Reply:
It does. Well spotted. I changed the code now so that if testsPhpVersion is set, xdebug will not work. I don't have the time to add compatibility for that, but at least if someone does specify this in their config, they will get that php version in tests.